docker-compose.yml

```diff
@@ -57,11 +55,11 @@ services:
       - S3_ENDPOINT=${S3_ENDPOINT}
       - S3_ACCESS_KEY=${S3_ACCESS_KEY}
       - S3_SECRET_KEY=${S3_SECRET_KEY}
+      - LOCAL_ROOT=/data
```

Comment:
if you're hardcoding the value of this env var, there is probably no point in having it be an env var. i would either let it be customisable in env, or just hardcode the string in the code

Reply:
This is due to me wanting to maintain the abillity for people to easily run RealistikGDPS outside of Docker. This is currently the best compromise I have come up with.

Reply:
but................. this value is not set in the env file meaning if they run it outside of docker it will literally crash currently

Reply:
`.env` file support is currently Docker only (through compose). Standalone only supports JSON and system environment variables.

Reply:
that is a skill issue you should seriously fix

Reply:
STC when we have live running instances ran by other people.

Reply:
disagree but i digress